docker-push.sh

```diff
@@ -7,5 +7,5 @@ VER="$(cat VERSION)"
 REV="$(git log -n 1 --no-walk --pretty="%h")"
 DATE="$(TZ="UTC" date "+%Y%m%d")"
 
-docker tag percona/pmm-server perconalab/pmm-server:"$VER-dev$DATE.$REV"
-docker push perconalab/pmm-server:"$VER-dev$DATE.$REV"
```

Comment:
this script created for copy containers from perconalab to percona organization.
so these changes broke this script